docs/development/affiliated-packages.rst

```diff
@@ -430,7 +430,8 @@ files manually`_ section since this explains what many of the files do.
 Updating to the latest template files
 -------------------------------------
 
-.. TODO
+See instructions in Item 14 above.
```

Comment:
Item 14 is this in the rendered HTML:
![screenshot](https://cloud.githubusercontent.com/assets/2090236/14985108/f1eadbd6-1113-11e6-93e2-6dd21bc75db3.png)